lib/node_modules/@stdlib/array/complex64/docs/types/index.d.ts

```diff
@@ -233,6 +233,33 @@ declare class Complex64Array implements Complex64ArrayInterface {
 	*/
 	get( i: number ): Complex64 | void;
 
+	/**
+	* Returns an array element at any integer index.
```

Comment:
```suggestion
	* Returns an array element.
```

It is fine that the description is the same as `#get`.